packages/@webex/plugin-meetings/src/media/MediaConnectionAwaiter.ts

```diff
@@ -83,6 +92,17 @@ export default class MediaConnectionAwaiter {
       `Media:MediaConnectionAwaiter#connectionStateChange --> connection state: ${this.webrtcMediaConnection.getConnectionState()}`
     );
 
+    if (this.isFailed()) {
```

Comment:
perhaps we should add some call analyzer metrics on this.

Reply:
we already send CA event "client.ice.end" with the appropriate error when we fail to establish the connection - see Meeting.waitForMediaConnectionConnected() - we will do it no matter if we get "failed" state from the browser or just "disconnected" or "connecting" and we eventually time out. I don't think sending any extra info to CA about the fact that we got "failed" from browser vs us just timing out waiting for "connected" is necessary, because it doesn't tell us any more information, it's just an implementation detail of the browser, for example Firefox very quickly  goes to "failed", while chrome usually stays in "disconnected" for a lot longer and we time out before it gets to "failed"